### DISCUSSION TITLE
How to get engine.http.response_rate ?

### DISCUSSION THREAD

**Discussion by yahakim419** [artilleryio/artillery#1917] [Artillery]
Hi,
i try to load my report from artillery to [https://github.com/artilleryio/report-viewer](url) but in my report i can't find "http.response_rate" so the graph cant work properly. I also curious why dated time can't read my periode.

Here my result :
<img width="1396" alt="Screen Shot 2023-05-17 at 23 22 51" src="https://github.com/artilleryio/artillery/assets/68334952/67267c58-253d-4e41-9251-33d2045ae1ca">
<img width="1416" alt="Screen Shot 2023-05-17 at 23 22 43" src="https://github.com/artilleryio/artillery/assets/68334952/fff5123e-571d-49b3-bda6-a211229b2811">

example report : 
<img width="453" alt="Screen Shot 2023-05-17 at 23 24 09" src="https://github.com/artilleryio/artillery/assets/68334952/667f18a8-f830-4475-adb5-481ec01ad034">

my report:
<img width="462" alt="Screen Shot 2023-05-17 at 23 25 00" src="https://github.com/artilleryio/artillery/assets/68334952/077142ce-1bec-4056-9978-471989718125">

Here my report files:
`{
  "aggregate": {
    "counters": {
      "vusers.created_by_name.find_cash_opname_by_branch": 198,
      "vusers.created": 198,
      "http.requests": 198,
      "http.codes.200": 198,
      "http.responses": 198,
      "vusers.failed": 0,
      "vusers.completed": 198
    },
    "rates": {
      "http.request_rate": 7
    },
    "firstCounterAt": 1684340474722,
    "firstHistogramAt": 1684340474991,
    "lastCounterAt": 1684340494336,
    "lastHistogramAt": 1684340494336,
    "firstMetricAt": 1684340474722,
    "lastMetricAt": 1684340494336,
    "period": 1684340490000,
    "summaries": {
      "http.response_time": {
        "min": 79,
        "max": 605,
        "count": 198,
        "p50": 104.6,
        "median": 104.6,
        "p75": 111.1,
        "p90": 125.2,
        "p95": 141.2,
        "p99": 459.5,
        "p999": 561.2
      },
      "vusers.session_length": {
        "min": 99.8,
        "max": 631.1,
        "count": 198,
        "p50": 127.8,
        "median": 127.8,
        "p75": 138.4,
        "p90": 159.2,
        "p95": 175.9,
        "p99": 487.9,
        "p999": 584.2
      }
    },
    "histograms": {
      "http.response_time": {
        "min": 79,
        "max": 605,
        "count": 198,
        "p50": 104.6,
        "median": 104.6,
        "p75": 111.1,
        "p90": 125.2,
        "p95": 141.2,
        "p99": 459.5,
        "p999": 561.2
      },
      "vusers.session_length": {
        "min": 99.8,
        "max": 631.1,
        "count": 198,
        "p50": 127.8,
        "median": 127.8,
        "p75": 138.4,
        "p90": 159.2,
        "p95": 175.9,
        "p99": 487.9,
        "p999": 584.2
      }
    }
  },
  "intermediate": [
    {
      "counters": {
        "vusers.created_by_name.find_cash_opname_by_branch": 15,
        "vusers.created": 15,
        "http.requests": 15,
        "http.codes.200": 12,
        "http.responses": 12,
        "vusers.failed": 0,
        "vusers.completed": 12
      },
      "rates": {
        "http.request_rate": 6
      },
      "firstCounterAt": 1684340474722,
      "firstHistogramAt": 1684340474991,
      "lastCounterAt": 1684340479938,
      "lastHistogramAt": 1684340479769,
      "firstMetricAt": 1684340474722,
      "lastMetricAt": 1684340479938,
      "period": "1684340470000",
      "http.request_rate": null,
      "summaries": {
        "http.response_time": {
          "min": 97,
          "max": 178,
          "count": 12,
          "p50": 104.6,
          "median": 104.6,
          "p75": 108.9,
          "p90": 133,
          "p95": 149.9,
          "p99": 149.9,
          "p999": 149.9
        },
        "vusers.session_length": {
          "min": 118.7,
          "max": 272.3,
          "count": 12,
          "p50": 135.7,
          "median": 135.7,
          "p75": 159.2,
          "p90": 169,
          "p95": 175.9,
          "p99": 175.9,
          "p999": 175.9
        }
      },
      "histograms": {
        "http.response_time": {
          "min": 97,
          "max": 178,
          "count": 12,
          "p50": 104.6,
          "median": 104.6,
          "p75": 108.9,
          "p90": 133,
          "p95": 149.9,
          "p99": 149.9,
          "p999": 149.9
        },
        "vusers.session_length": {
          "min": 118.7,
          "max": 272.3,
          "count": 12,
          "p50": 135.7,
          "median": 135.7,
          "p75": 159.2,
          "p90": 169,
          "p95": 175.9,
          "p99": 175.9,
          "p999": 175.9
        }
      }
    }
  ]
}`

**Comment by yahakim419** [comment]
my setup :
VERSION INFO:

Artillery: 2.0.0-32
Node.js:   v19.6.0
OS:        darwin

**Reply by hassy** [reply]
hi @yahakim419 👋 That report viewer is no longer supported (apologies for not having the repository marked as such). Artillery has a built-in `artillery report` command which can be used to generate self-contained HTML reports.

**Reply by yahakim419** [reply]
Ah, i see. Thank you.
